client/http4s-client/src/main/scala/sttp/tapir/client/http4s/EndpointToHttp4sClient.scala

```diff
@@ -147,6 +149,9 @@ private[http4s] class EndpointToHttp4sClient(clientOptions: Http4sClientOptions)
       case RawBodyType.InputStreamBody =>
         val entityEncoder = EntityEncoder.inputStreamEncoder[F, InputStream]
         req.withEntity(Applicative[F].pure(encoded.asInstanceOf[InputStream]))(entityEncoder)
+      case RawBodyType.InputStreamRangeBody =>
+        val entityEncoder = EntityEncoder.inputStreamEncoder[F, InputStream]
+        req.withEntity(Sync[F].delay(encoded.asInstanceOf[InputStreamRange].inputStream()))(entityEncoder)
```

Comment:
hm or even `.blocking`?

Reply:
Hmm I thought it will be only reading bytes from memory, but actually we may want to avoid such an assumption. I'll use blocking just in case.